dracut/30afterburn/afterburn-hostname.service

```diff
@@ -16,4 +16,6 @@ OnFailureJobMode=isolate
 
 [Service]
 ExecStart=/usr/bin/afterburn --cmdline --hostname=/sysroot/etc/hostname
+ExecStart=/bin/sh -c 'mkdir -p /sysroot/run/tmpfiles.d'
```

Comment:
Also, maybe throw a comment to e.g. https://github.com/coreos/ignition/issues/635? (Or a more targeted issue if there is one).

Reply:
Will do

Reply:
Heh, I meant a comment in this file to that GitHub issue, but consider it optional.